src/Symfony/Component/Console/Application.php

```diff
@@ -1086,19 +1087,6 @@ public function setDefaultCommand($commandName)
         $this->defaultCommand = $commandName;
     }
 
-    private function stringWidth($string)
-    {
-        if (!function_exists('mb_strwidth')) {
```

Comment:
It already did: https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Console/Helper/Helper.php#L52-L63

The two methods were strictly the same (and changes in upper branches were reported on both identically)